Nona-Creative/nona-playground
# feat(aws-cdk-s3-cloudfront): s3 and cloudfront static site deployment package

**Scope**

https://nonaredteam.atlassian.net/browse/PT-53

**Work done**

Created package for a stack that will deploy a static site to s3 and serve from cloudfront, with integration in Route 53

**Steps to test**

- Run the test suite
- Deploy example app in the package folder to playgrounds aws account (see readme in src/packages/aws-cdk-s3-cloudfront for instructions)
